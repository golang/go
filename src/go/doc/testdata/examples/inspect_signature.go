// Copyright 2021 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package foo_test

import (
	"bytes"
	"io"
)

func getReader() io.Reader { return nil }

func do(b bytes.Reader) {}

func Example() {
	getReader()
	do()
	// Output:
}

func ExampleIgnored() {
}
