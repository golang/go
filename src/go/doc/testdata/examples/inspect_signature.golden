-- .Play --
package main

import (
	"bytes"
	"io"
)

func getReader() io.Reader { return nil }

func do(b bytes.Reader) {}

func main() {
	getReader()
	do()
}
-- Ignored.Play --
package main

import ()

func main() {
}

